test/zksync-tests/CompactSignature.test.ts

```diff
@@ -0,0 +1,40 @@
+import {ethers} from "ethers";
+import {expect} from "chai";
+
+
+
+function getCompactSignature(vRaw: number, rRaw: string, sRaw: string): [string, string] {
+    const v = vRaw - 27;
+    const vs = ethers.utils.hexZeroPad(
+        ethers.BigNumber.from(v).shl(255).add(ethers.BigNumber.from(sRaw)).toHexString(),
+        32
+    );
+    return [rRaw, vs];
+}
+
+describe('CompactSignature', function () {
+
+    it("should verify compact signature with v=27", async () => {
+        const r = "0x68a020a209d3d56c46f38cc50a33f704f4a9a10a59377f8dd762ac66910e9b90";
+        const s = "0x7e865ad05c4035ab5792787d4a0297a43617ae897930a6fe4d822b8faea52064";
+        const v = 27;
+
+        const vs = getCompactSignature(v, r, s);
```

Comment:
What you are checking in this test? 

In the original tests, they tested that Solidity `_getCompactSignature` function works as intended. Here you don't even call Solidity contract that implemented the method, but instead just test your own typescript implementation of the method